### PR TITLE
Release updates from dev branch.

### PR DESCRIPTION
**Pull request by kaiwalyajoshi**

### CODE REVIEW

**Comment by kaiwalyajoshi** [comment]
This PR incorporates @akirillov's fixes to `kubeaddons-tests` from this PR: https://github.com/mesosphere/kubeaddons-tests/pull/20

File: Dispatchfile [hunk 1]
```diff
@@ -1,7 +1,64 @@
...
+                  ssh-keyscan -t rsa github.com > /etc/ssh/known_hosts
+                  git config --global url."https://$GITHUB_TOKEN:@github.com/".insteadOf "https://github.com/"
+                  echo $PWD
+                  ls -a
+                  make kind-test
+                  """
+              ],
+              resources = k8s.corev1.ResourceRequirements(
+              limits = {
+                "cpu": k8s.resource_quantity("8000m"),
```
Comment (alejandroEsc) [inline Dispatchfile:38]: Why 8 cpu and 6 gi of memory?

**Review by alejandroEsc — COMMENTED** [review]
integration test is failing. Question about the how much cpu and memory the integration test needs, do we have metrics as to why? are we oom? Finally we should keep CI changes and addons on different PRs

File: Dispatchfile (hunk 1 above)
Comment (kaiwalyajoshi) [reply Dispatchfile:38]: The TC test is the required one. We'll fix the Dispatch related issues in subsequent PRs.

**Review by akirillov — DISMISSED** [review]
Approving as TC test is passing.

**Review by joejulian — APPROVED** [review]
@faiq might want to look at fixing this repo to use the cluster claim controller

**Review by joejulian — APPROVED** [review]

**Review by akirillov — APPROVED** [review]

**Pull request merged**